src/js/pages/universe/PackagesTab.js

```diff
@@ -13,9 +13,18 @@ import FilterInputText from '../../components/FilterInputText';
 import Image from '../../components/Image';
 import InstallPackageModal from '../../components/modals/InstallPackageModal';
 import Loader from '../../components/Loader';
+import Page from '../../components/Page';
 import Panel from '../../components/Panel';
 import StringUtil from '../../utils/StringUtil';
 
+const UniverseBreadcrumbs = () => {
+  const crumbs = [
+    <Link to="universe/packages" key={-1}>Universe</Link>
```

Comment:
Going back to John's comment, do we want to change it to `Packages` so it matches with the Sidebar link?